DevelopGitHub/GitHubExtension/GitHubExtension.Security.WebApi.Library/SimpleInjectorConfiguration.cs

```diff
@@ -29,7 +22,9 @@ public static Container RegisterPackages(Container container)
             container.RegisterPackages(new[]
             {
                 typeof(DALPackage).Assembly,
-                typeof(WebApiLibraryPackage).Assembly
+                typeof(WebApiLibraryPackage).Assembly,
+                typeof(NotesDALPackage).Assembly,
```

Comment:
This particular package was at a start point of my task and I only added typeof(NotesDALPackage).Assembly,
typeof(NotesWebApiPackage).Assembly
But our team will note this remark and rename packages to more specific names